refinery/core/models.py

```diff
@@ -833,6 +833,13 @@ def is_valid(self):
         else:
             return False
 
+    def get_nodes(self, uuids_only=False):
```

Comment:
It's probably best to avoid flag arguments in favor of separate functions, especially if the return type is different (like in this case)
https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior
https://martinfowler.com/bliki/FlagArgument.html